samples/tutorials/mnist/01_Lightweight_Python_Components.ipynb

```diff
@@ -0,0 +1,635 @@
+    "    divmod_task = divmod_op(add_task.output, b, '/')\n",
+    "\n",
+    "    #For an operation with a multiple return values, the output references can be accessed using `task.outputs['output_name']` syntax\n",
+    "    result_task = add_op(divmod_task.outputs['quotient'], c)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Compile the pipeline"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pipeline_func = calc_pipeline\n",
+    "pipeline_filename = pipeline_func.__name__ + '.pipeline.zip'\n",
+    "import kfp.compiler as compiler\n",
+    "compiler.Compiler().compile(pipeline_func, pipeline_filename)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "00. Setup.ipynb\r\n",
+      "01. Local Development Quickstart.ipynb\r\n",
+      "02. Lightweight Python components.ipynb\r\n",
+      "03. GCP Components Pipeline.ipynb\r\n",
+      "04. TFX Pipeline from Pre-build Components.ipynb\r\n",
+      "05. Spark and XGboost Pipeline.ipynb\r\n",
+      "calc_pipeline.pipeline.zip\r\n",
+      "mlpipeline-metrics.json\r\n",
+      "mlpipeline-ui-metadata.json\r\n"
+     ]
+    }
+   ],
+   "source": [
+    "! ls ."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Submit the pipeline"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 29,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "Experiment link <a href=\"https://kubeflow1.endpoints.kubeflow-pipeline-fantasy.cloud.goog/pipeline/#/experiments/details/1062f310-3760-4e35-a208-c81363e501e9\" target=\"_blank\" >here</a>"
+      ],
+      "text/plain": [
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "text/html": [
+       "Run link <a href=\"https://kubeflow1.endpoints.kubeflow-pipeline-fantasy.cloud.goog/pipeline/#/runs/details/14261ce8-0655-4275-960e-88018f50fb18\" target=\"_blank\" >here</a>"
+      ],
+      "text/plain": [
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "#Specify pipeline argument values\n",
+    "arguments = {'a': '7', 'b': '8'}\n",
+    "\n",
+    "experiment = client.create_experiment('python-functions')\n",
+    "\n",
+    "#Submit a pipeline run\n",
+    "run_name = pipeline_func.__name__ + ' run'\n",
+    "run_result = client.run_pipeline(experiment.id, run_name, pipeline_filename, arguments)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Train a keras model"
```

Comment:
Maybe briefly introduce what this model does?

Reply:
Makes sense. Added an introduction.